packages/mgt-chat/src/statefulClient/StatefulGraphChatClient.ts

```diff
@@ -1008,9 +1012,40 @@ detail: ${JSON.stringify(eventDetail)}`);
     return result;
   }
 
-  private buildAcsMessage(graphMessage: ChatMessage, currentUser: string, messageId: string, content: string): Message {
+  private hasUnsupportedContent(content: string, attachments: ChatMessageAttachment[]): boolean {
+    const unsupportedContentTypes = [
+      'application/vnd.microsoft.card.codesnippet',
+      'application/vnd.microsoft.card.fluid',
+      'reference'
+    ];
+    const isUnsupported: boolean[] = [];
+
+    if (attachments.length) {
+      for (const attachment of attachments) {
+        const contentType = attachment?.contentType ?? '';
+        isUnsupported.push(unsupportedContentTypes.includes(contentType));
+      }
+    } else {
+      const unsupportedContentRegex = /<\/?[atchmenbl]+>/gim;
```

Comment:
I think that this is too broad given that a reasonable plain HTML mesage as reported by @sebastienlevert is being reported as unsupported.